### PR TITLE
Update assets and extension [Deploy] Deploy 28/11/2022

### PR DESCRIPTION
**Pull request by ghost**
## WHAT

1. Update Assets, png to svg
2. Update constants 
3. Update Extension
4. Fix file use Extension

## Evidence (Screenshot or Video)

- N/A

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/Zts4P8 |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Message | Sun* commit message | Template: [TASK/FREATURE/BUG/SUPPORT][RedmineID] + message |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: Whollet/Whollet/Source/Presentation/Address/AddressViewController.swift [hunk 1]
```diff
@@ -97,7 +97,7 @@ final class AddressViewController: UIViewController {
     @IBAction func sendICXTap(_ sender: UIButton) {
         if let address =  self.addressTextField.text, address != "" {
             self.appDelegate?.transactionInput.address = address
-            // TODO: Implement to confirm page
+            self.navigationController?.pushViewController(ConfirmationViewController.instantiate(), animated: true)
```
Comment (tunghs-1253) [inline Whollet/Whollet/Source/Presentation/Address/AddressViewController.swift:100]: Clean architecture của Tú không chia thêm Navigator riêng à :v

Comment (ghost) [reply Whollet/Whollet/Source/Presentation/Address/AddressViewController.swift:100]: Không á anh ới, bữa sợ sai có gì chậm deadline nên mấy cái đó cái nào làm rồi em lấy làm lại thôi á :3 có gì qua project 2 em update thêm kiến trúc

File: Whollet/Whollet/Source/Core/Constants/AppConstants.swift [hunk 2]
```diff
@@ -29,6 +29,15 @@ struct AppConstants {
         static let failedGetCamera = "Failed to get the camera device"
         static let iconShow = "show"
         static let iconHide = "hide"
+        static let allTransaction = "All transaction"
+        static let sent = "sent"
+        static let sentTitle = "Sent"
```
Comment (ghost) [inline Whollet/Whollet/Source/Core/Constants/AppConstants.swift]: duplicate

File: Whollet/Whollet/Source/Core/Constants/AppConstants.swift [hunk 3]
```diff
@@ -29,6 +29,15 @@ struct AppConstants {
...
+        static let sendHeader = "Transaction confirmed"
+        static let rejected = "rejected"
+        static let rejectedTitle = "Rejected"
```
Comment (ghost) [inline Whollet/Whollet/Source/Core/Constants/AppConstants.swift]: as above

**Pull request merged**